src/lib/service/deploy/sfdx.js

```diff
@@ -132,12 +132,18 @@ const
 			allScratchOrgs = _.get(config, 'sfdx.scratchOrgs'),
 			scratchOrgs = _.map(allScratchOrgs, org => ({ name: org.username, value: org }));
 
+		let defaultValue = 0;
+
 		if (config.options && config.options.includeNew && config.options.includeNew.sfdx === true) {
 			scratchOrgs.push(newOrg);
 		}
 
+		if (_.get(config, 'orizuru.sfdx.org.username')) {
+			defaultValue = _.indexOf(_.map(allScratchOrgs, org => org.username), config.orizuru.sfdx.org.username);
```

Comment:
Looks like a job for [findIndex](https://lodash.com/docs/4.17.4#findIndex)

Reply:
We'll do a code sweep after Dreamforce to see what we can improve 👍